### DISCUSSION TITLE
Bootstrap v5 new file path

### DISCUSSION THREAD

**Discussion by el0i404** [twbs/bootstrap#34403] [General]
Hi,

I _upgraded_ Bootstrap v4.6.0 to v5.0.2. 

In v4.6.0 I'm using the following files from _node_modules_ folder, but in v5.0.2 those files do not exist.

```
@import '~bootstrap/scss/mixins/text-emphasis';
@import '~bootstrap/scss/mixins/text-hide';
@import '~bootstrap/scss/mixins/hover';
```

[~bootstrap/scss/mixins v5](https://github.com/twbs/bootstrap/tree/main/scss/mixins)

Would like to know if there is a kind of document (changelog e.g) which shows where this path is pointing to so I can keep using the same code.

Thanks

**Comment by fchiumeo** [comment]
https://getbootstrap.com/docs/5.0/migration/

**Reply by el0i404** [reply]
Hi @fchiumeo, thanks for you answer and the link.

I've already check it and trying to find for example:

```
@mixin hover() {
  &:hover { @content; }
}
```

in another file, but I could not.

So that's means that the code _snippet_ above does not exist anymore in `Bootstrap v5` in any file? 🤔
